website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -165,7 +165,29 @@ For detailed instructions explaining how to build and run a pipeline that uses M
 
 ## Beam Java SDK support
 
-RunInference API is available to Beam Java SDK 2.41.0 and later through Apache Beam's [Multi-language Pipelines framework](https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines). Please see [here](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/transforms/RunInference.java) for the Java wrapper transform to use and please see [here](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/test/java/org/apache/beam/sdk/extensions/python/transforms/RunInferenceTransformTest.java) for some example pipelines.
+The RunInference API is available with the Beam Java SDK versions 2.41.0 and later through Apache Beam's [Multi-language Pipelines framework](https://beam.apache.org/documentation/programming-guide/#multi-language-pipelines). For information about the Java wrapper transform, see [RunInference.java](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/transforms/RunInference.java). For example pipelines, see [RunInferenceTransformTest.java](https://github.com/apache/beam/blob/master/sdks/java/extensions/python/src/test/java/org/apache/beam/sdk/extensions/python/transforms/RunInferenceTransformTest.java).
+
+## TensorFlow support
+
+To use TensorFlow with the RunInference API, create a model handler from within `tfx_bsl`. The model handler can be keyed or unkeyed.
+For more information, see [run_inference.py](https://github.com/tensorflow/tfx-bsl/blob/d1fca25e5eeaac9ef0111ec13e7634df836f36f6/tfx_bsl/public/beam/run_inference.py) in the TensorFlow GitHub repository.
+
+```
+tf_handler = CreateModelHandler(inference_spec_type)
```

Comment:
In this example it may make sense to add some information about importing the correct libs from tfx-bsl.

Probably the best way to get this done accurately would be to make a simple notebook and then import the code from that here.

I can try to make some time this week to do that.  Probably it's possible to just modify some code in the notebook you created that showed using the tfx-bsl interface and just replace that with the beam interface.

Reply:
I added an example below based on the email thread. Should I link to [this notebook](https://colab.sandbox.google.com/drive/1WwxXRnCAd13wEUEBWvuyEI_spnMHbIJK?resourcekey=0-G_S520h-ED_cRHKjLA9GUw&usp=sharing) in this section so that customers can use it to import the code from?

Reply:
We will need to link the official notebook linked from github. But we can do this as a separate PR as that will take a while to get done.